zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/AuthorizationController.java

```diff
@@ -47,6 +48,20 @@ public CompletableFuture<ResponseEntity<Object>> createAuthorization(
         .fold(RestErrorMapper::mapProblemToCompletedResponse, this::create);
   }
 
+  @CamundaGetMapping(path = "/authorizations/{authorizationKey}")
```

Comment:
🔧 We should probably put `/authorizations` to the `@RequestMapping` on the class. Now we are adding it to every path in this controller.